packages/blade/src/components/Button/BaseButton/buttonTokens.ts

```diff
@@ -69,21 +69,61 @@ const buttonPadding: Record<
   },
 };
 
+const buttonIconOnlyPadding: Record<
+  NonNullable<BaseButtonProps['size']>,
+  Record<'top' | 'bottom' | 'left' | 'right', keyof Theme['spacing']>
+> = {
+  xsmall: {
+    top: 0,
+    bottom: 0,
+    left: 2, // should be `6px` as per design but we're making it `8px` since `6px` is not available as a spacing token
+    right: 2, // should be `6px` as per design but we're making it `8px` since `6px` is not available as a spacing token
```

Comment:
Should we add the missing spacing token?

Reply:
Not yet. This will be a longer exercise since it `6px` sits between 4 & 8 which would mean changing the spacing tokens and ensuring we change it everywhere 😅

Reply:
Spoke to Saurav, we shall keep it 8px here but pick this refactor up later